### DISCUSSION TITLE
A Gentle Introduction to Formal Verification

### DISCUSSION THREAD

**Discussion by subbdue** [subbdue/systemverilog.io#6] [Website Comments]
Please use this Discussions thread for questions and comments on the article `A Gentle Introduction to Formal Verification`

**Comment by ManuelSOliveira** [comment]
Hi, I would propose to update the links for the following papers:

[Design Guidelines for Formal Verification](https://dvcon-proceedings.org/wp-content/uploads/design-guidelines-for-formal-verification.pdf)

[Guidelines for creating a formal verification testplan](https://www.researchgate.net/publication/228360702_Guidelines_for_creating_a_formal_verification_testplan)

Excellent Work!

**Reply by subbdue** [reply]
Hi @ManuelSOliveira, Thanks for this! I've updated the website.
